src/scripts/content/reviewboard.js

```diff
@@ -0,0 +1,17 @@
+/*jslint indent: 2 */
+/*global $: false, document: false, togglbutton: false*/
+'use strict';
+
+togglbutton.render('#content_container:not(.toggl)', {observe: true}, function () {
```

Comment:
This element appears in the root of all documents. Should select an element that's related to what's being tracked, perhaps `.review-request`?

Reply:
Yes, that would be an improvement.